migration/src/m0000572_sbom_package_license.rs

```diff
@@ -0,0 +1,63 @@
+use crate::m0000015_create_license_category_enums::LicenseCategory;
+use crate::UuidV4;
+use sea_orm_migration::prelude::*;
+
+#[derive(DeriveMigrationName)]
+pub struct Migration;
+
+#[async_trait::async_trait]
+impl MigrationTrait for Migration {
+    async fn up(&self, manager: &SchemaManager) -> Result<(), DbErr> {
+        manager
+            .create_table(
+                Table::create()
+                    .table(SbomPackageLicense::Table)
+                    .col(
+                        ColumnDef::new(SbomPackageLicense::Id)
+                            .uuid()
+                            .not_null()
+                            .default(Func::cust(UuidV4))
+                            .primary_key(),
+                    )
+                    .col(ColumnDef::new(SbomPackageLicense::SbomId).uuid().not_null())
+                    .col(
+                        ColumnDef::new(SbomPackageLicense::LicenseId)
+                            .uuid()
+                            .not_null(),
+                    )
+                    .col(
+                        ColumnDef::new(SbomPackageLicense::NodeId)
+                            .string()
+                            .not_null(),
+                    )
+                    .col(ColumnDef::new(SbomPackageLicense::LicenseType).enumeration(
+                        LicenseCategory::LicenseCategory,
+                        [
+                            LicenseCategory::SLC,
+                            LicenseCategory::SLD,
+                            LicenseCategory::O,
+                        ],
+                    ))
+                    .to_owned(),
+            )
+            .await?;
```

Comment:
Since these records are owned by sboms, there should be a cascade delete added to delete these records when the sbom is deleted.

Reply:
@bxf12315 please merge the referenced PR from Hiram.